### DISCUSSION TITLE
23.10

### DISCUSSION THREAD

**Discussion by Sunderland93** [Ubuntu-Sway/Ubuntu-Sway-Remix#46] [Announcements]
This is a latest stable release of Ubuntu Sway Remix 23.10 "Mantic Minotaur". You can download ISO from the official website: https://ubuntusway.com/downloads.php

What's new:

- Integrated [sway-systemd](https://github.com/alebastr/sway-systemd), which allows to run user daemons and applications as systemd services. Most default applications, such as Waybar or poweralertd now runing as systemd user services;
- EralyOOM daemon is replaced by systemd-oomd, which, thanks to sway-systemd, now correctly identifies the most resource-intensive processes;
- rofi-bluetooth script is replaced by [blutuith](https://github.com/darkhz/bluetuith) manager;
- **lxpolkit-gtk3** is replaced by **mate-polkit**;
- Small improvements in Waybar panel;
- Added support for Raspberry Pi 5;
- Updated packages: sway-input-config 1.3.0, nwg-look 0.2.2, nwg-drawer 0.3.9, nwg-displays 0.3.7, nwg-bar 0.1.3, musikcube 3.0.

<hr /><em>This discussion was created from the release <a href='https://github.com/Ubuntu-Sway/Ubuntu-Sway-Remix/releases/tag/23.10'>23.10</a>.</em>

**Comment by devobern** [comment]
Is there a way to update my current installation, or do I have to install the new version from scratch?

**Comment by Sunderland93** [comment]
I recommend installing from scratch. Release upgrade is not properly testing and may cause various issues
